vllm/model_executor/layers/linear.py

```diff
@@ -741,12 +744,12 @@ def weight_loader_v2(self,
                          loaded_shard_id: Optional[str] = None):
         if loaded_shard_id is None:  # special case for certain models
```

Comment:
Can you name the certain models you know of so we can have an idea of a type?

Reply:
Yes will add comments